YAML/sublime-syntax.sublime-syntax

```diff
@@ -0,0 +1,532 @@
+        2: punctuation.separator.key-value.yaml
+        3: keyword.other.omap.yaml
+        4: punctuation.definition.keyword.yaml
+
+    - match: (\&|\*)\w.*?$
+      scope: variable.other.yaml
+      captures:
+        1: punctuation.definition.variable.yaml
+
+    - match: '(\<\<): ((\*).*)$'
+      scope: keyword.operator.merge-key.yaml
+      captures:
+        1: entity.name.tag.yaml
+        2: keyword.operator.merge-key.yaml
+        3: punctuation.definition.keyword.yaml
+
+    - match: '-'
+      scope: punctuation.section.begin.yaml
+
+    - match: (#).*?(?=%>)
+      scope: punctuation.section.begin.yaml
+
+    - match: '[^''"%\-:?@`&*!,#|>\s\[\]{}](?:[^\[\]{}:#,]|:[^ ]|(?<! )#)*'
+      scope: string.unquoted.yaml
+
+  variables_:
+    - meta_scope: meta.variables
+    - include: comments
+    - match: '^(?=\w)'
+      pop: true
+
+    # block regex
+    - match: '^(\s*)([\w_]*)\s*(:)\s*(>-|\|-|\|)'
+      captures:
+        2: entity.name.function.declaration
+        3: punctuation.separator.key-value
+        4: punctuation.definition.block.begin
+      push:
+        - meta_content_scope: string.unquoted.yaml
+        - match: ^(?!^\1\s\s)
+          pop: true
+        # - include: regex_unquoted_line
+        - include: regex_unquoted
+
+    # inline regex
+    - match: '\s*([\w_]+)\s*(:)\s*'
+      captures:
+        1: entity.name.function.declaration
+        2: punctuation.separator.key-value
+      push: regex
+
+  contexts_:
+    - meta_scope: meta.contexts
+    - include: comments
+    - match: '^(?=\w)'
+      pop: true
+
+    # Keyword literals
+    - match: \b(true|false|null)\s*(?=$|,|\}|\])
+      scope: constant.yaml
+
+    # block regex
+    - match: '(^\s*)(-)\s+(match)\s*(:)\s*(>-|\|-|\|)'
+      captures:
+        2: punctuation.section.begin.yaml
+        3: meta.tag.yaml entity.name.tag
+        4: punctuation.separator.key-value
+        5: punctuation.definition.block.begin
+      push:
+        - meta_content_scope: string.unquoted.yaml
+        - match: ^(?!^\1\s\s\s\s)
+          pop: true
+        - include: regex_unquoted
+      # push:
+      #   - meta_scope: string.unquoted.block.yaml
+      #   - match: '^(?!^\1)|^(?=\1(-|(?:  )?\w+\s*:)|#)'
+      #     pop: true
+      #   - match: $
+      #   - include: regex_unquoted_line
+
+    # inline regex
+    - match: '(-)\s+(match)\s*(:)\s*'
+      captures:
+        1: punctuation.section.begin.yaml
+        2: meta.tag.yaml entity.name.tag
+        3: punctuation.separator.key-value
+      push:
+        - match: $
+          pop: true
+        - include: comments
+        - include: regex
+
+    # captures
+    - match: '^(\s*)(captures)\s*(:)\s*'
+      captures:
+        2: keyword.control.flow.syntax
+        3: punctuation.separator.key-value
+      push:
+        - meta_content_scope: meta.capturing.yaml
+        - include: comments
+        - match: '^(\s*)(\d+)\s*(:)'
+          captures:
+            2: support.variable
```

Comment:
for my use, I changed it from `support.variable` to `constant.numeric`, so that it would be highlighted by the color scheme :)